redundant-bmc/src/services_impl.hpp

```diff
@@ -41,9 +41,12 @@ class ServicesImpl : public Services
     /**
      * @brief Returns this BMC's position.
      *
-     * @return - The position
+     * @param[in] silent - If error traces should be printed
```

Comment:
Is this "silent" concept new? Why wouldn't we want to print out error messages in both cases?

Reply:
Yeah, I'm really not a fan of passing around booleans and doing if/elses on them (maybe it's from my fsp-trace days). If anything we could do a marcro wrapper around them that hides it, but it's error path and a debug tool so I don't see anything wrong with just printing it out.

Reply:
done.